let sayHello = new Promise(function (resolve, reject) {
    reject('unable to say hi');
    setTimeout(function () {
        resolve('hi, universe');
    }, 5000);
});

sayHello
    .then(function (msg) {
        console.log('ðŸš€ ~ msg:', msg);
    })
    .catch(function (error) {
        console.warn('ðŸš€ ~ error:', error);
    });

console.warn('show this line of errror');
